llvm/test/CodeGen/AMDGPU/sdwa-peephole-instr-combine-sel2.mir

```diff
@@ -0,0 +1,796 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1030 -simplify-mir -run-pass=si-peephole-sdwa -o - %s | FileCheck %s
+
+# Test the combination of SDWA selections in si-peephole-sdwa. In each
```

Comment:
Do we have tests for combining on dst?

Reply:
No. I considered this to "demonstrate all different selection combinations" as I wrote in another comment and those are the same for the source and destination selections. Do you think we need to add a dedicated test for the dst selections?

Reply:
The destination case is significantly more likely to break, so should also cover those

Reply:
I had added a test for the dst_sel values that I saw being introduced in `SIPeepholeSDWA::matchSDWAOperands`, i.e. one of the values always has to come from a non-SDWA instruction and I think I cannot introudce arbitrary selection values in this way.